sqrl-tools/sqrl-run/src/main/java/com/datasqrl/DatasqrlRun.java

```diff
@@ -112,33 +138,65 @@ public CompiledPlan compileFlink() {
     config.putIfAbsent("table.exec.source.idle-timeout", "1 s");
     config.putIfAbsent("taskmanager.network.memory.max", "800m");
     config.putIfAbsent("execution.checkpointing.interval", "30 sec");
+    config.putIfAbsent("execution.checkpointing.min-pause", "20 s");
     config.putIfAbsent("state.backend", "rocksdb");
     config.putIfAbsent("table.exec.resource.default-parallelism", "1");
+    config.putIfAbsent("execution.target", "remote");
+    config.putIfAbsent("rest.address", "localhost");
 
     Configuration configuration = Configuration.fromMap(config);
-    StreamExecutionEnvironment sEnv = StreamExecutionEnvironment.createLocalEnvironmentWithWebUI(configuration);
+    String udfJarPath = "/Users/henneberger/sqrl/sqrl-testing/sqrl-integration-tests/src/test/resources/udf/build/deploy/flink/lib/myjavafunction-0.1.0-snapshot-all.jar";
+//    String sqrlRunJarPath = "/Users/henneberger/sqrl/sqrl-tools/sqrl-run/target/sqrl-run.jar";
+
+    URL[] urls = {
+        new File(udfJarPath).toURI().toURL(),
+//        new File(sqrlRunJarPath).toURI().toURL()
+    };
+    URLClassLoader urlClassLoader = new URLClassLoader(urls);
+
+
+    //todo: check to see if execution.target is local or remote. If local, use getExe
+    String mode = config.get("execution.target");
+    StreamExecutionEnvironment sEnv;
+    switch (mode){
+      case "local":
+        //todo udfs?
+        sEnv = StreamExecutionEnvironment.getExecutionEnvironment();
```

Comment:
local is flink's minicluster for some int tests, which would assume UDFs are in the classpath

Reply:
For example, it'll be able to read our libraries that we imported as mvn dependencies but not externally downloaded jars. Remote will use externally downloaded dependencies but requires a new system lib folder.

Reply:
We can unify it further by doing some clever deps arranging, but right now it covers all of our bases.